src/components/Map/MapContainer.tsx

```diff
@@ -121,6 +116,8 @@ export const MapContainer = () => {
 
   const submitInputs = async () => {
     toggleLoading(true);
+    setIsExpanded(!isExpanded);
+    toggleExpandedResults(!expandedResults);
```

Comment:
I know these are probably defaulted to something but don't you think it would be clearer to set them to an absolute value, like `setIsExpended(true)` since we know when they submit we want to expand?

Reply:
That is what I thought too at first, but the label on the expand button does not change to minimize results when we do `setIsExpanded(true)`.